metakb/schemas.py

```diff
@@ -487,3 +487,159 @@ def schema_extra(schema: Dict[str, Any],
                     }
                 ]
             }
+
+
+class SearchID(BaseModel):
+    """Queries for the Search by ID Endpoint."""
+
+    node_id: Optional[str]
+
+    class Config:
+        """Configure examples."""
+
+        @staticmethod
+        def schema_extra(schema: Dict[str, Any],
+                         model: Type['SearchID']) -> None:
+            """Configure OpenAPI schema"""
+            if 'title' in schema.keys():
+                schema.pop('title', None)
+            for prop in schema.get('properties', {}).values():
+                prop.pop('title', None)
+            schema['example'] = {
+                "node_id": "civic:vid33"
+            }
+
+
+class SearchIDService(BaseModel):
+    """Define model for Search by ID Endpoint Response."""
+
+    query: SearchID
```

Comment:
You do not need a class for a single string. This string needs to be required. 
```suggestion
    query: str
```